packages/monitoring/LICENSE

```diff
@@ -0,0 +1,201 @@
+                            Apache License
```

Comment:
We can ditch this file. It is copied from the root of the repo before we publish the module to npm.